database_management_bot.py

```diff
@@ -0,0 +1,2235 @@
+        parse_mode="MarkdownV2",
+        reply_markup=reply_markup,
+    )
+
+    return CONTACT
+
+
+@under_maintenance
+def get_admin_photo(update: Update, context: CallbackContext) -> int:
+    """
+    The get_admin_photo_experimental function is called when the user sends a contact.
+    It stores the phone number in context.user_data, and then asks for a selfie photo to be sent.
+
+    :param update: Update: Access the telegram api
+    :param context: CallbackContext: Send data between functions
+    :return: The next state of the concersation
+    """
+    logger.info("User send contact")
+
+    reply_keyboard = [['Скасувати реєстрацію']]
+
+    context.user_data["phone_number"] = update.message.contact.phone_number
+    logger.info("Storing admin phone number")
+
+    context.user_data["ADMIN_INFO"] = update.message.contact.to_dict()
+    logger.info("Storing admin contact info")
+
+    update.message.reply_text(
+        text='👤 *Тепер, надішліть, будь ласка, фото свого обличчя*'
+             '\n\n😃 Не лякайтесь\! Ця процедура потрібна для підвищення безпеки під час війни\.'
+             '\n\n🔒 Ми подбаємо про *конфіденційність та безпеку* ваших даних\.'
+             ' Уся інформація зберігається на *захищених серверах*\. Ми *не будемо поширювати* її\.'
+             '\n\n❕ Будь ласка, використовуйте реальну фотографію вашого обличчя\.'
+             ' Фотографуйте при хорошому освітленні та старайтесь, щоб ваше обличчя було добре видно на фото\.'
+             '\n\n❗️ Вам буде заблоковано доступ за використання несправжніх фотографій обличчя\.',
+        parse_mode="MarkdownV2",
+        reply_markup=ReplyKeyboardMarkup(
+            reply_keyboard,
+            one_time_keyboard=True,
+            resize_keyboard=True,
+            input_field_placeholder='Надішліть фото',
+        ),
+    )
+
+    return ADMIN_PHOTO
+
+
+@under_maintenance
+def add_admin(update: Update, context: CallbackContext) -> ConversationHandler.END:
+    """
+    The add_admin function checks whether user sent correct photo. If there are no faces or too many faces on photo
+    it asks user to send it agait and returns the previous state of the conversation.
+    If everything is fine it adds a new admin to the database.
+    It takes in an update and context objects as arguments, and returns ConversationHandler.END
+    to end the conversation after it is completed.
+
+    :param update: Update: Pass the incoming update
+    :param context: CallbackContext: Store data on the user's conversation flow
+    :return: Conversationhandler.END
+    """
+    user_id = update.effective_user.id
+    user = update.message.from_user
+
+    reply_keyboard = MAIN_REPLY_KEYBOARD
+
+    if update.message.photo:
+        logger.info("User sent photo")
+
+        id_img = update.message.photo[-1].file_id
+        photo = context.bot.getFile(id_img)
+        image_bytes = io.BytesIO()
+        photo.download(out=image_bytes)
+    else:
+        return
```

Comment:
What if the user sends not a photo but, for example, text?

Reply:
This issue is now mentioned in issue #7 and will be fixed in future alpha updates.